catalog_update/catalog_item.py

```diff
@@ -246,12 +250,12 @@ def upgrade(self) -> dict:
             image['tag'] = value['latest_tag']
 
         with open(os.path.join(self.path, summary['upgrade_details']['filename']), 'w') as f:
-            f.write(yaml.safe_dump(values, sort_keys=False, indent=2))
+            self.YAML.dump(values, f)
 
         test_values_path = os.path.join(self.path, summary['upgrade_details']['test_filename'] or '')
         if summary['upgrade_details']['test_filename'] and os.path.exists(test_values_path):
             with open(os.path.join(test_values_path), 'r') as f:
-                test_values = yaml.safe_load(f.read())
+                test_values = self.YAML.load(f.read())
```

Comment:
With the new CI we don't use `test_values.yaml` anymore. Maybe we should remove this?
(also there is no image versions in the `ci/*-values.yaml` now, it uses the image version specified in the `values.yaml`)